docs/platforms/android/index.mdx

```diff
@@ -121,11 +129,15 @@ import io.sentry.Sentry
 class MyActivity : AppCompatActivity() {
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
+    // Any code running after this line will be
+    //profiled until Sentry.stopProfiler() is called.
+    Sentry.startProfiler()
     try {
       throw Exception("This is a test.")
     } catch (e: Exception) {
       Sentry.captureException(e)
     }
+    Sentry.stopProfiler()
```

Comment:
`m`: I think we should also add/remove this depending on whether the profiling feature is selected

Reply:
👍 
i'll take a note and do it later